frontend/src/toolbar/flags/featureFlagsLogic.ts

```diff
@@ -97,6 +105,10 @@ export const featureFlagsLogic = kea<featureFlagsLogicType>({
             ;(window['posthog'] as PostHog).onFeatureFlags((_, variants) => {
                 toolbarLogic.actions.updateFeatureFlags(variants)
             })
+            const locallyOverrideFeatureFlags = (window['posthog'] as PostHog).get_property('$override_feature_flags')
```

Comment:
We should expose this property from `posthog-js` through something like `posthog.feature_flags.hasLocalOverrides()`. I couldn't get my dev env for `posthog-js` running though...